Assets/_Project/Scripts/Network/ParamaterPass.cs

```diff
@@ -0,0 +1,9 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public static class ParamaterPasss
```

Comment:
Should be `ParameterPass`, but a more descriptive name would be something like `ConnectionInfo`.  `ParameterPass` is very vague